Contributing.md

```diff
@@ -0,0 +1,88 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+# Pull Request Process
+1.	Fork the repository and create a separate branch regarding your contribution.
+2.	Update the Changelog file (Changes to README file should be commited **ONLY** when there are changes relevant to the essence of the project, or the ways to deploy it!) with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations, and container parameters.
+3.	Make sure the pull request passes all checks
+4.	You may merge the Pull Request once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+5.  Ensure that your code follows the PEP 8 Style guide for Python code before submitting a pull request.
+We have also set up Flake8 to ensure that our style guides are being followed(PR's that dont pass this may not get reviewed until those are solved)
```

Comment:
Note that it is expected that when this PR is merged it is expected that flake8 does setup up.
Otherwise it may be misleading for the contributors.
There are two options here:
- Wait for #10 to merge
- Omit the part about flake8, and add it at #10 PR.